source/cloud/azure/azureml.md

```diff
@@ -254,8 +258,12 @@ sweep_job = command_job_for_sweep.sweep(
 
 # submit job
 returned_sweep_job = ml_client.create_or_update(sweep_job)
+returned_sweep_job
 ```
 
+Once the job is created, click on the details page provided in the output of `returned_sweep_job`, or go
```

Comment:
Oh great, the repr of the object gives you a link to the page?

Reply:
Yes, to the specific job page under the experiment.